core/templates/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -97,6 +100,9 @@ angular.module('oppia').directive('storyNodeEditor', [
             $scope.isStoryPublished = StoryEditorStateService.isStoryPublished;
             $scope.currentTitle = $scope.nodeIdToTitleMap[$scope.getId()];
             $scope.editableTitle = $scope.currentTitle;
+            $scope.currentDescription =
```

Comment:
Use `(..)` and break after `(` here.

Reply:
I had removed it as requested by different comment.